cypress/e2e/FooterNavigation.cy.js

```diff
@@ -0,0 +1,14 @@
+import Home from '../src/pages/Home';
+import Footer from '../src/components/Footer';
+
+const homePage = new Home();
+const footer = new Footer();
+
+describe('Footer navigation tests', () => {
+    it('Check links on Follow us section in footer', () => {
+        homePage
+            .visit()
+            .checkPageUrl();
+           footer.checkFooterLinks();
+    });
+});
```

Comment:
Why do you need FooterNavigation.cy.js file? I see that you already have the navigation.cy.js file. Also, why do name this file with class naming conventions?